src/Installers/Windows/WindowsHostingBundle/SharedFramework.wxs

```diff
@@ -24,40 +24,29 @@
             <RollbackBoundary Id="RB_SHAREDFX_REDIST_BUNDLE" />
 
             <!-- OPT_NO_SHAREDFX could be unset at this point, which we explicitly treat as 'false' -->
-            <ExePackage Id="SharedFxRedist_arm64" SourceFile="$(var.InstallersOutputPath)\$(var.SharedFxRedistInstallerarm64)"
+            <MsiPackage Id="SharedFxRedist_arm64" SourceFile="$(var.InstallersOutputPath)\$(var.SharedFxRedistInstallerarm64)"
                         Name="$(var.SharedFxRedistInstallerarm64)"
                         Compressed="yes"
                         Vital="yes"
                         InstallCondition="(NativeMachine=&quot;$(var.NativeMachine_arm64)&quot;) AND (NOT OPT_NO_SHAREDFX OR OPT_NO_SHAREDFX=&quot;0&quot;)"
-                        InstallCommand="/quiet /norestart"
-                        RepairCommand="/quiet /norestart /repair"
-                        Permanent="yes"
-                        DetectCondition="SharedFxRedistProductVersion_arm64 = v$(var.SharedFxInstallerProductVersionarm64)">
-            </ExePackage>
+                        Permanent="yes">
```

Comment:
Think we need to remove the Permanent attribute too. When set, the MSI is marked as a system component so it won't remove it.

Reply:
Done